lib/elixir/lib/calendar/date.ex

```diff
@@ -83,7 +86,7 @@ defmodule Date do
       -366
```

Comment:
Can we add a space after this as well, since we're doing it in a couple other places in this PR?